### DISCUSSION TITLE
LightColorBody Voreinstellung für Visu

### DISCUSSION THREAD

**Discussion by flkontakt** [Zefau/ioBroker.jarvis#2220] [Q&A]
Hallo zusammen,

siehe Bild, kann man die Kachelauswahl als Standardansicht einstellen? 
![Screenshot_20230205_170151_Chrome~2](https://user-images.githubusercontent.com/77227687/216830387-f6414970-b55a-4044-b0a1-04480353edea.jpg)

**Comment by Zefau** [comment]
Aktuell geht das nicht. Du kannst aber gerne einen Feature Request aufmachen.
